dip-0003.md

```diff
@@ -329,9 +323,9 @@ The new system removes the need for masternode reward payment votes and allows t
 The new rules to determine the next block’s payee are:
 
 1. Take the last block's payee. If it is a regular masternode (type 0), go to step 5.
-2. If the last HPMN payee was already paid 4 blocks in a row, go to step 5.
-3. If the last HPMN is not present in the current masternode list anymore, go to step 5.
-4. Select last HPMN payee as the next block's payee. Return.
+2. If the last EvoNode (type 1) payee was already paid 4 blocks in a row, go to step 5.
+3. If the last EvoNode is not present in the current masternode list anymore, go to step 5.
+4. Select last EvoNode payee as the next block's payee. Return.
```

Comment:
Let's handle these renames in a separate PR since they're not directly related to this. See #163